pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "peewee-model-utils"
+version = "0.1.0"
+description = ""
+authors = ["rmondal00 <rahul.mondal@ingrammicro.com>"]
+readme = "README.md"
+packages = [{include = "common_model_peewee_utils"}]
```

Comment:
Why this package ? your root package is model_utils.
Anyway I would suggest to follow our namespace package rules and put all these stuffs within

`connect.utils.peewee` package similar to 
https://github.com/cloudblue/connect-markdown-renderer/tree/master/connect/utils/terminal/markdown